shared/cfg/base.cfg

```diff
@@ -528,6 +528,10 @@ used_mem = 512
 # Cpu model params
 auto_cpu_model = "yes"
 cpu_model_flags = ""
+pseries:
+    # Configure cpu_model
+    auto_cpu_model = "no"
+    cpu_model = host
```

Comment:
May I know why these two config only works for pseries?

Reply:
Avocado-vt can't provide a cpu model for PowerPC, so I added `pseries:` to always use the "host" model.